configure.ac

```diff
@@ -23,16 +23,28 @@ AC_ARG_ENABLE([coverage],
     )
 )
 if test "$enable_coverage" = "yes"; then
+    AC_SUBST([GO_COVERAGE_ENABLED], [yes])
     AC_SUBST([GO_COVERAGE_ARG], [-cover])
+
+    AC_SUBST_FILE([MAKE_COVERAGE_TARGETS])
+    MAKE_COVERAGE_TARGETS=$srcdir/Makefile.coverage_targets
```

Comment:
While having the ability to optionally create these targets in the Makefile is neat, I don't see any harm in having them there.

The error you'd get when trying to run them without having coverage enabled would probably be easily explainable?

Reply:
I'm actually thinking about removing these targets completely. They don't fit into our CI story, because we run the system tests in distributed fashion, so we'll have to collate the coverage data in some other way anyway.

Reply:
Targets from this file were removed completely because they were not useful for collecting coverage data and generating report in our parallelized CI workflow. Their functionality was moved to the github workflow file.

Reply:
The line is still there though, should it not be removed now that the `Makefile.coverage_targets` is no longer in use?